### ISSUE TITLE
She went and never came back

### ISSUE THREAD

**Issue by umdoisumdois** [Albeoris/Memoria#487]
Freya jumped after getting hit, but before taking damage, and never came back down
I waited till 15 mins left, but I screenshotted sooner
Speed = 5 in the config
![20240519212252_1](https://github.com/Albeoris/Memoria/assets/153212487/d9a9b2c6-5565-4c2c-a60a-19a327db22b4)

**Comment by snouz** [comment]
Sorry to change your beautiful title :/

**Comment by snouz** [comment]
@Tirlititi

**Comment by Tirlititi** [comment]
Ah, I wonder if she got confused by Tantarian's attack and that's what caused the bug.
Do you have any gameplay mod activated, Alternate Fantasy maybe? If that's so, was it "Edge" the attack that Tantarian used on Freya when she jumped, or was it Paper Storm?

**Comment by umdoisumdois** [comment]
No, I'm playing with Memoria only, and the attack used was Edge
